README.md

```diff
@@ -131,12 +137,48 @@ Same as `prop`, except that it returns `props.theme[path]` instead of
 #### Examples
 
 ```javascript
+import styled from "styled-components";
+import { theme } from "styled-tools";
+
 const Button = styled.button`
  color: ${theme("button.color", "red")};
 `;
 ```
 
-Returns **PropsWithThemeFn** 
+### palette
+
+Returns `props.theme.palette[key][index]` or `defaultValue`.
```

Comment:
as per the sample below, this should be `props.theme[key][index]` as in `props.theme["primary"][1]` (there's no `palette` key in the theme object)